test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/Internal/ChunkVisitorTestFactory.cs

```diff
@@ -44,6 +44,7 @@ public static IList<Chunk> GetTestChunks(bool visitedTagHelperChunks)
                 GetTagHelperChunk("Baz"),
                 GetNestedViewComponentTagHelperChunk("Foo", visitedTagHelperChunks),
                 GetViewComponentTagHelperChunk("Bar", visitedTagHelperChunks),
+                GetIndexerViewComponentTagHelperChunk("Bee", visitedTagHelperChunks, "Dictionary<string, List<int>>"),
```

Comment:
Just so the generated output is correct, update the attribute type name to be the full name representation.